lib/Installation/Partitioner/ExpertPartitionerPage.pm

```diff
@@ -141,4 +149,39 @@ sub press_ok_clone {
     send_key($self->{ok_clone_shortcut});
 }
 
+sub modify_create_new_partition_table {
```

Comment:
Why not just create_new_partition_table ?

Reply:
Because it's after selecting hard disks and pressing Modify->create new partition table, there is also the option to select particular disk and then click Expert->create new partition table

Reply:
that is the exactly the confusion I think, it should be just `modify_hard_disks`, meaning just clicking on modify when I'm on Hard Disks (disks in plural) as there is another way to access to the same thing as you mentioned, so we remove that ambiguity, WDYT? (also we need to exclude last too lines in another method as indicated below)

Reply:
someone with more than one disk in the future might need the version in singular with exactly the same shortcut but from my point of view that is fine to duplicate it in the code. in fact I'm no sure if you have more than one hd you might not be offered that option at that point unless it will create the same partition table for all the disk (I didn't test that, not sure)

Reply:
Hmm... I am not sure if "modify_hard_disks" is clear enough, because my function particularly presses "Modify" button and then "Create new partition table..". There is also the option "Modify"->"Edit Hard Disk". I have tried this sequence with 2 hard disks, it still works in the same way.